src/features/cleaner_file_type.rs

```diff
@@ -53,3 +67,63 @@ pub fn directory_cleaner_based_on_file_type(
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::fs::{self, File};
```

Comment:
self?

Reply:
Just a matter of convenience, it allows to access module functions directly without using full path